pkg/build/Godeps/Godeps.json

```diff
@@ -0,0 +1,205 @@
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/homedir",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/idtools",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/ioutils",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/jsonlog",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/jsonmessage",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/longpath",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/pools",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/promise",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/stdcopy",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/system",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/term",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/docker/pkg/term/windows",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-5445-ga2bf823",
+			"Rev": "a2bf823a94f77bb041a5ef336c689adb09a593b6"
+		},
+		{
+			"ImportPath": "github.com/docker/go-connections/nat",
+			"Comment": "v0.2.0-2-gf549a93",
+			"Rev": "f549a9393d05688dff0992ef3efd8bbe6c628aeb"
+		},
+		{
+			"ImportPath": "github.com/docker/go-units",
+			"Comment": "v0.3.1-10-ge30f1e7",
+			"Rev": "e30f1e79f3cd72542f2026ceec18d3bd67ab859c"
+		},
+		{
+			"ImportPath": "github.com/fsouza/go-dockerclient",
+			"Comment": "0.2.1-1283-ga45f2c4",
+			"Rev": "a45f2c476754603041aeb54751207d5e5ec30b77"
+		},
+		{
+			"ImportPath": "github.com/fsouza/go-dockerclient/testing",
+			"Comment": "0.2.1-1283-ga45f2c4",
+			"Rev": "a45f2c476754603041aeb54751207d5e5ec30b77"
+		},
+		{
+			"ImportPath": "github.com/gorilla/context",
+			"Rev": "215affda49addc4c8ef7e2534915df2c8c35c6cd"
+		},
+		{
```

Comment:
If you dump fsouza/go-dockerclient/testing you can remove these.  Do you need testing?